components/infra-proxy-service/server/orgs_test.go

```diff
@@ -290,8 +290,6 @@ func TestOrgs(t *testing.T) {
 			})
 			require.NoError(t, err)
 			require.NotNil(t, orgs)
-			assert.Contains(t, orgs.Orgs, resp1.Org)
```

Comment:
I have removed these checks all over, planning to work on adding more test cases in separate PR.